protege-desktop/pom.xml

```diff
@@ -151,7 +151,7 @@
 
 					<plugin>
 						<artifactId>maven-assembly-plugin</artifactId>
-						<version>2.4</version>
+						<version>3.6</version>
```

Comment:
There is no such version of the `maven-assembly-plugin`. The correct version number is `3.6.0`.